terraform/source_test.go

```diff
@@ -54,12 +54,41 @@ func TestSplitSourceUrl(t *testing.T) {
 			actualRootRepo, actualModulePath, err := SplitSourceUrl(sourceUrl, terragruntOptions.Logger)
 			require.NoError(t, err)
 
-			assert.Equal(t, testCase.expectedRootRepo, actualRootRepo.String())
+			assert.Equal(t, testCase.expectedSo, actualRootRepo.String())
 			assert.Equal(t, testCase.expectedModulePath, actualModulePath)
 		})
 	}
 }
 
+func TestToSourceUrl(t *testing.T) {
+	t.Parallel()
+
+	testCases := []struct {
+		sourceURL         string
+		workingDir        string
```

Comment:
looks like `workingDir` is empty in each test case, do we need it in the struct?

Reply:
Thanks. Added a few test cases with the workingDir values.